int32 arm_type
int32 WHOLE_BODY = 0
int32 LEFT_ARM = 1
int32 RIGHT_ARM = 2
int32 LEFT_ARM_WITHOUT_WAIST = 12
int32 RIGHT_ARM_WITHOUT_WAIST = 22
int32 BOTH_ARM = 3
int32 LEFT_GRIPPER = 4
int32 RIGHT_GRIPPER = 8
int32 BOTH_GRIPPER = 16
int32 MOBILE_BASE = 32

int32 plan_mode
int32 IK_MODE = 1
int32 JOINT_MODE = 2

sensor_msgs/JointState start_state
geometry_msgs/PoseStamped pose

---
trajectory_msgs/JointTrajectory trajectory
